chef/cookbooks/neutron/templates/default/l3_agent.ini.erb

```diff
@@ -50,7 +50,6 @@ handle_internal_only_routers = <%= @handle_internal_only_routers %>
 # empty value for the linux bridge. when this parameter is set, each L3 agent
 # can be associated with no more than one external network.
 # external_network_bridge = br-ex
-external_network_bridge = <%= @external_network_bridge %>
```

Comment:
For me it is ok look comment above `when this parameter is set, each L3 agent can be associated with no more than one external network`